port/raspberrypi/rp2xxx/src/hal/pio/assembler/tokenizer.zig

```diff
@@ -2278,3 +2278,15 @@ test "tokenize.instr.comment with no whitespace" {
         .delay = .{ .expression = "1" },
     }, tokens.get(0));
 }
+
+test "format tokenizer" {
+    const test_tokenizer = Tokenizer(.RP2040).init("out 1");
+    const string = try std.fmt.allocPrint(std.testing.allocator, "{}", .{test_tokenizer});
+    defer std.testing.allocator.free(string);
+    try std.testing.expectEqualStrings(
```

Comment:
Looks like we already assign this a short name
```suggestion
    try expectEqualStrings(
```